mycpp/gc_str.h

```diff
@@ -74,6 +75,10 @@ class Str {
     return ObjHeader::Str();
   }
 
+  int hash(HashFunc h);
+
+  static constexpr int kHashFlagMask = 0x08000000;
```

Comment:
why this value?  Needs a comment

Reply:
oh, it's just the most significant bit. gonna inline this, though